### ISSUE TITLE
Field not working when in a repeater

### ISSUE THREAD

**Issue by romaincazier** [neuerituale/FieldtypeOembed#2]
Hi,

First of all thank you for your module.

I'm having an issue where I have my field in a repeater matrix but it prints out the error "No oembed data or provider found.". However the same link in a field not being in a repeater works without any issues.

I wonder if being in a repeater gives it a special context and somehow avoids your module from working properly ?

Let me know if you need more informations.

**Comment by romaincazier** [comment]
I did some digging and looking for the error message I came across the `render` function of the Inputfield and my issue seems to be because of [line 41](https://github.com/neuerituale/FieldtypeOembed/blob/da6071b59c194204b7d86be29afe10b000d52c43/InputfieldOembed.module#L41), more specifically with `getFormatted`.

I don't know why but in the context of a repeater `getFormatted` returns `null`.

Also a small note : `$this->name` returns the field name suffixed with `_repeater{id}`, but here it's not helpful since even when inputing `$fieldtype->name` it still returns `null`.

Hope it helps.
